### PR TITLE
Update learn-hyperbridge.md

### PR DESCRIPTION
**Pull request by chungquantin**
Supersedes https://github.com/w3f/polkadot-wiki/pull/5474

### CODE REVIEW

File: docs/learn/learn-hyperbridge.md [hunk 1]
```diff
@@ -6,3 +6,78 @@ description: Polkadot's Bridge to Ethereum.
 keywords: [bridge, polkadot, ethereum]
 slug: ../learn-hyperbridge
 ---
+
+import RPC from "./../../components/RPC-Connection";
+
+:::info Learn about Parachain and Bridges
+
+To fully follow the material on this page, it is recommended to be familiar with the concepts of the [Parachain](https://wiki.polkadot.network/docs/learn-parachains) and [Bridges](./learn-bridges.md).
```
Comment (filippoweb3) [inline docs/learn/learn-hyperbridge.md]:
```suggestion
To fully follow the material on this page, it is recommended to be familiar with the concepts of [Parachain](./learn-parachains) and [Bridges](./learn-bridges.md).
```

File: docs/learn/learn-hyperbridge.md [hunk 2]
```diff
@@ -6,3 +6,78 @@ description: Polkadot's Bridge to Ethereum.
...
+
+:::
+
+Interoperability is the core vision of the Polkadot technology. Through years of blockchain development, there has been many effort into making a secure interoperability solution to communicate between blockchains.
```
Comment (filippoweb3) [inline docs/learn/learn-hyperbridge.md]:
```suggestion
Interoperability is the core vision of the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} technology. Through years of blockchain development, much effort has been put into making a secure interoperability solution between blockchains.
```

File: docs/learn/learn-hyperbridge.md [hunk 3]
```diff
@@ -6,3 +6,78 @@ description: Polkadot's Bridge to Ethereum.
...
+
+With that vision in mind, Hyperbridge (short for hyper-scalable bridge) is innovated as a cross-chain solution built as an interoperability coprocessor. Hyperbridge is crafted to scale cryptographically secure, consensus, and state-proof-based interoperability across all blockchains. Before diving deeper into the core technology of the Hyperbridge, let's go through a first few foundational concepts.
```
Comment (filippoweb3) [inline docs/learn/learn-hyperbridge.md]:
```suggestion
Hyperbridge (short for hyper-scalable bridge) is innovated as a cross-chain solution built as an interoperability coprocessor. Hyperbridge is crafted to scale cryptographically secure, consensus, and state-proof-based interoperability across all blockchains.
```

File: docs/learn/learn-hyperbridge.md [hunk 4]
```diff
@@ -6,3 +6,78 @@ description: Polkadot's Bridge to Ethereum.
...
+
+Ensuring secure cross-chain communication involves the meticulous verification of various aspects, including: [Consensus Mechanisms](https://wiki.polkadot.network/docs/learn-consensus), [Consensus Faults](https://research.polytope.technology/consensus-proofs), [State Proofs](https://research.polytope.technology/state-machine-proofs) and [State Transitions](https://wiki.polkadot.network/docs/learn-parachains#state-transitions).
+
+> What is a coprocessor?
+>
+> **Coprocessor**, in the context of hardware, can be referred to a microprocessor designed to supplement the capabilities of the primary processor. For example, GPU is a coprocessor of the CPU to be optimized for graphical and simultaneous computation.
+
+Due to the complexity and expensiveness of the onchain verification process, in coprocessor model, the computation is performed off-chain and the outcomes of the execution, along with cryptographic proofs validating their accuracy, are subsequently presented on-chain.
+
+Expanding more about the coprocessor model, it has been applied in other solutions of offloading cryptographic computation as well, particularly, **SNARK coprocessor model**.
```
Comment (filippoweb3) [inline docs/learn/learn-hyperbridge.md]: any link to SNARK coprocessor model?

File: docs/learn/learn-hyperbridge.md [hunk 5]
```diff
@@ -6,3 +6,78 @@ description: Polkadot's Bridge to Ethereum.
...
+
+Interoperable State Machine Protocol (ISMP) provides a familiar HTTP-like API for developers who want to make cross-chain requests to trigger certain logic on the counterparty chain. It allows making POST requests to send arbitrary data to connected chains, as well as GET requests to read the storage (verified through state proofs) of applications on connected chains.
+
+In addition to facilitating cross-chain message passing among connected chains, ISMP also serves as a synchronisation primitive across Hyperbridge's internal state machines. This enables its parachain cores to communicate with each other and delegate tasks.
+
+### Underlying technologies
+
+Underlying technologies of the Hyperbridge is integrated with:
+
+- **PLONK verifier**
```
Comment (filippoweb3) [inline docs/learn/learn-hyperbridge.md]: link?

File: docs/learn/learn-hyperbridge.md [hunk 6]
```diff
@@ -6,3 +6,78 @@ description: Polkadot's Bridge to Ethereum.
...
+- [BEEFY consensus](https://spec.polkadot.network/sect-finality#sect-grandpa-beefy)
+- The Barretenberg backend
```
Comment (filippoweb3) [inline docs/learn/learn-hyperbridge.md]: link? Or brief explanation?

File: docs/learn/learn-hyperbridge.md [hunk 7]
```diff
@@ -6,3 +6,78 @@ description: Polkadot's Bridge to Ethereum.
...
+
+This capability enables the distribution of the validation workload for consensus, state proofs, and state transition re-execution across various designated [Parachain Cores](https://github.com/polkadot-fellows/RFCs/blob/6f29561a4747bbfd95307ce75cd949dfff359e39/text/0001-agile-coretime.md). Hence, Polkadot is utilized by the Hyperbridge as a verifiable computation layer to provide the ["Full Node Security"](https://blog.polytope.technology/introducing-hyperbridge-interoperability-coprocessor#full-node-level-security) in cross-chain bridges.
+
+Hence, an additional layer of security is provided, allowing Hyperbridge to detect and prevent [Byzantine](https://en.wikipedia.org/wiki/Byzantine_fault) behaviors across connected chains.
+
+### Interoperable State Machine Protocol (ISMP)
+
+Interoperable State Machine Protocol (ISMP) provides a familiar HTTP-like API for developers who want to make cross-chain requests to trigger certain logic on the counterparty chain. It allows making POST requests to send arbitrary data to connected chains, as well as GET requests to read the storage (verified through state proofs) of applications on connected chains.
```
Comment (filippoweb3) [inline docs/learn/learn-hyperbridge.md]: Link to ISMP?

**Review by filippoweb3 — CHANGES_REQUESTED** [review]
Thank you for this great contribution to the Wiki! I added some feedback.

**Comment by chungquantin** [comment]
@filippoweb3 thanks for your feedback. I'll will make changes based on it

**Comment by filippoweb3** [comment]
@chungquantin there are still some links missing. After that I am happy to approve and the PR.

**Comment by chungquantin** [comment]
@filippoweb3 I have added links to resolve your feedback. Please take a look and let me know if there are anything else to update.

**Review by filippoweb3 — APPROVED** [review]
thank you! I will create a separate PR to do some minor edits and add you as a reviewer :)

**Pull request merged**